torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1307,6 +1340,16 @@ def create_symbolic_sizes_strides_storage_offset(self, ex: torch.Tensor, source:
                 )
         assert all(x is not None for x in stride)
         sym_size = [self.create_symintnode(i, hint=hint) for i, hint in zip(size, ex.size())]
+
+        for i, syms in enumerate(sym_size):
```

Comment:
nit: syms implies it is plural, but it is not

Reply:
Agreed